packages/algorand-js/test/integration/escrow-account.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint sonarjs/no-duplicate-string: 0 */
+import { assert } from "chai";
+
+import { ERRORS } from "../../src/errors/errors-list";
+import { Runtime } from "../../src/index";
+import { StoreAccountImpl } from "../../src/runtime/account";
+import { getAcc } from "../helpers/account";
+import { expectTealErrorAsync } from "../helpers/errors";
+import { johnAccount } from "../mocks/account";
+
+describe("Algorand Stateless Smart Contracts", function () {
+  const escrow = new StoreAccountImpl(1000000000); // 1000 ALGO
+  const john = new StoreAccountImpl(500, johnAccount); // 0.005 ALGO
+  // set up transaction paramenters
+  const txnParams = {
+    type: 0, // payment
```

Comment:
please update this to use `enum`

Reply:
I am getting this error if i use `enum` in ts:
```
Argument of type '{ type: TransactionType; sign: number; fromAccount: Account; toAccountAddr: string; amountMicroAlgos: number; payFlags: { totalFee: number; }; }' is not assignable to parameter of type 'AlgoTransferParam | AssetTransferParam | SSCCallsParam | ExecParams[]'.
  Type '{ type: TransactionType; sign: number; fromAccount: Account; toAccountAddr: string; amountMicroAlgos: number; payFlags: { totalFee: number; }; }' is not assignable to type 'AlgoTransferParam'.
    Types of property 'type' are incompatible.
      Type 'TransactionType' is not assignable to type 'TransactionType.TransferAlgo'
```

Reply:
checking

Reply:
```
import { TransactionType, SignType, AlgoTransferParam } from "algob/src/types";
const txnParams: AlgoTransferParam = {
    type: TransactionType.TransferAlgo, // payment
    sign: SignType.SecretKey,
    fromAccount: escrow.account,
    toAccountAddr: john.address,
    amountMicroAlgos: 100,
    payFlags: { totalFee: 1000 }
  };
```

Reply:
this works can you please check ?

Reply:
works, thanks. (We can use `ExecParams` as a general type for txnParams for all tests)